src/pipelines/worldpop/worldpop_population.py

```diff
@@ -0,0 +1,101 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import zipfile
+from io import BytesIO
+from functools import partial
+from typing import Any, Dict, List, Tuple
+
+import requests
+from tqdm import tqdm
+from pandas import DataFrame, Series, read_csv, isnull
+
+from lib.concurrent import thread_map
+from lib.net import download
+from lib.pipeline import DataSource
+from lib.cast import age_group
+
+
+class WorldPopPopulationDataSource(DataSource):
+    """ Retrieves the requested properties from Wikidata for all items in metadata.csv """
```

Comment:
What part of wikidata? A more descriptive docstring would probably be warranted here, along with maybe an example URL if possible.

Reply:
This is factually wrong, actually. I updated the comment.